src/pages/App.svelte

```diff
@@ -27,27 +27,17 @@
   let showAbout = false;
   let showInstructions = false;
 
-  // TODO This is a hacky way of detecting environment. 'npm run dev' and
```

Comment:
It's also nice to get rid of this distinction. This code path isn't testable locally, so that's a bit scary anyway!